osu.Desktop/Program.cs

```diff
@@ -23,7 +23,16 @@ public static int Main(string[] args)
             // Back up the cwd before DesktopGameHost changes it
             var cwd = Environment.CurrentDirectory;
 
-            using (DesktopGameHost host = Host.GetSuitableHost(@"osu", true))
+            string gameName = @"osu";
+
+            if (DebugUtils.IsDebugBuild)
+            {
+                var customNameArg = args.SingleOrDefault(s => s.StartsWith(@"--name=", StringComparison.Ordinal));
```

Comment:
`--name` is a really bad (read: nondescript) choice.

Not really sure if we should be changing the complete game name here, either. Is there a reason you didn't focus directly on the storage rather than the whole game?

Reply:
Yeah the name of the argument was very arbitrary, maybe `--game-identifier` or something? not sure.

I made this argument change the complete game name as I think it's the best choice to define a whole different place and data for a game instance that can be run side-by-side with the main one.

It seems to be potentially possible to modify `OsuStorage` for that, but I'm not really sure if it's worth adding further logic in the main game codebase for "debug builds"-specific features, rather than going with this simple way of changing the game name.

Unless you think there could potentially be an issue with changing the game name?

Reply:
The only way I'd be happy with this happening is if you make it into an `int` only suffix:

`--debug-client-id=1` - starts osu! with identifier `osu-1`